src/IconButton/IconButton.scss

```diff
@@ -0,0 +1,41 @@
+$iconbutton-hover-size: 1.6em;
+
+.iconbutton-hover {
+    @include button-focus(black);
+    width: $iconbutton-hover-size;
+    height: $iconbutton-hover-size;
+    border-radius: $iconbutton-hover-size;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    position: relative;
+
+    &:hover{
```

Comment:
```suggestion
    &:hover {
```

Reply:
Oh, ha, we should maybe have a sass linter...